### PR TITLE
ZEN-17164: Dropzone - Add blocks variant

### PR DESCRIPTION
**Pull request by chavlji**
Addition to this PR:
https://github.com/reciprocity/zenlib/pull/93

Adds .blocks variant of dropzone where instructions aren't inlined:
Screenshot 2020-09-29 at 11 46 52

versus inlined variant:
Screenshot 2020-09-29 at 11 47 15

### CODE REVIEW

**Comment by zenbot-0** [comment]
🔬 [Click here](https://reciprocity.github.io/zenlib/branches/17164-Filedropzone-blocks) to test your changes on Storybook

**Review by leferreyra — APPROVED** [review]
Looks good!

Would be great if both variants were available on storybook.
Could you add font awesome to storybook so we can see the icons?

**Comment by mplesec** [comment]
@chavlji @leferreyra a question tho, since these are suppose to be standalone components, shouldn't we add font awesome as a dependency to this component?

**Comment by leferreyra** [comment]
Yep. At least as a peer dependency

**Comment by chavlji** [comment]
Light svg icons added. It works in storybook and it should be ok now.

File: README.md [hunk 1]
```diff
@@ -27,29 +27,40 @@ To browse a list of available components go to [Reciprocity's NPM page](https://
...
 
-### Run storybook locally
+In the second terminal window go to `./packages/_storybook_` and run:
+1. `npm run storybook`
 
-To run the storybook website locally you can do it by running
+Storybook now runs on `http://localhost:6006/` and all changes you make to any package/component are immediatelly reflected in the storybook.
 
-`npm run storybook`
+#### Componetn changes reflection in Storybook: In-depth
```
Comment (mplesec) [inline README.md:43]: typo: Componetn

File: README.md [hunk 2]
```diff
@@ -27,29 +27,40 @@ To browse a list of available components go to [Reciprocity's NPM page](https://
...
+Zenlib is a multi-npm-package repo (based on [Lerna](https://github.com/lerna/lerna)). Each component is it's own npm package.
+
+Command `./npm run bootstrap` is cruical here. It searches for all inter-repo package dependencies and replaces those packages with a symbolic link that shows directly to `dist` folder of each package.
```
Comment (mplesec) [inline README.md:46]: typo: cruical

File: README.md [hunk 3]
```diff
@@ -27,29 +27,40 @@ To browse a list of available components go to [Reciprocity's NPM page](https://
...
+
+So whenever `dist` folder of a package is rebuilt due to some changes (due to running `build:watch` command), this change will get picked up by a Storybook and you'll be able to see the changes immediatelly.
 
 #### Building packages
 
-After you change stuff in the component's source code, you'll need to build the packages so they're distributable. You can do that by running `npm run build`. Or if you want files to be watched so they build automatically after you make a change, you'll want to run `npm run build:watch` instead.
+After you change stuff in the component's source code, you'll need to build the packages so they're distributable. You can do that by running `npm run build`.
 
-_The storybook website will also watch for changes while it's running. So if you want to develop using hot reloading, you need to run the storybook website in one terminal, and `npm run build:watch` in another one._
+But since you're probabbly already running `npm run build:watch`, which rebuilds package on each change, there's no need to do this manual build.
```
Comment (mplesec) [inline README.md:56]: typo: probabbly

File: packages/file-dropzone/FileDropzone.vue [hunk 4]
```diff
@@ -56,6 +59,11 @@ export default {
       ];
     }
   },
+  mounted() {
+    // Add fa replace watcher:
```
Comment (mplesec) [inline packages/file-dropzone/FileDropzone.vue:63]: Why is this needed?

**Review by mplesec — APPROVED** [review]
Awesome 😎 Fix the typos in a followup PR if you want, but this is ready to merge.

File: packages/file-dropzone/FileDropzone.vue (hunk 4 above)
Comment (chavlji) [reply packages/file-dropzone/FileDropzone.vue:63]: To replace <i> with the actual <svg>. That's the official way to include just some subset of icons. https://fontawesome.com/how-to-use/on-the-web/advanced/svg-javascript-core

File: README.md (hunk 3 above)
Comment (chavlji) [reply README.md]: I've fixed all typos in the readme.

**Review by mplesec — APPROVED** [review]
